docs/rules/no-this-before-super.md

```diff
@@ -1,12 +1,12 @@
-# Disallow to use `this`/`super` before `super()` calling in constructors. (no-this-before-super)
+# Disallow use of `this`/`super` before calling `super()` in constructors. (no-this-before-super)
 
-On the constructor of derived classes, if `this`/`super` are used before `super()`, it raises a reference error.
+In the constructor of derived classes, if `this`/`super` are used before `super()` calls, it raises a reference error.
 
 This rule checks `this`/`super` keywords in constructors, then reports those that are before `super()`.
 
 ## Rule Details
 
-This rule is aimed to flag `this`/`super` keywords before `super()` callings.
+This rule is aimed at flagging `this`/`super` keywords before `super()` callings.
```

Comment:
Heh, I wanted to make a comment about exactly this line in the original pull request. But went and searched the Internet for `aimed at vs. aimed to` and apparently `aimed to` is more correct form, even though `aimed at` sounds much more natural to me.